animeal/src/Flows/Main/Modules/HomeModule/Home/View/HomeViewController.swift

```diff
@@ -173,14 +173,19 @@ private extension HomeViewController {
         var closestLocation: CLLocation?
         var smallestDistance: CLLocationDistance?
 
+        let minPermittedDistinationInMeters: CLLocationDistance = 10000
```

Comment:
Is this change related to the logout toast?

Reply:
Removed, will be added in separate PR